integration-tests/feature_list.txt

```diff
@@ -20,8 +20,8 @@ features/stack_analysis_pypi_direct_and_transitive.feature
 features/stack_analyses_maven_ecosystem.feature
 features/stack_analysis_maven_direct_and_transitive.feature
 features/stack_analyses_npm_ecosystem.feature
-features/stack_analysis_npm_direct_and_transitive.feature
-features/stack_analyses_smoke_tests_pypi_ecosystem.feature
+#features/stack_analysis_npm_direct_and_transitive.feature
+#features/stack_analyses_smoke_tests_pypi_ecosystem.feature
```

Comment:
ecosystem_pypi_pylist.feature and smoke_tests_pypi were same but one contains SLA/SLO checks other doesn't so merged them by adding SLA/SLO checks in the ecosystem feature and commented out the other feature